### PR TITLE
L3 Combined Risks design improvements

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-7354

### CODE REVIEW

File: packages/frontend/src/components/host-chain-risks-warning.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  hostChain: ScalingProjectDisplay
+  riskCount?: number
+}
+
+export function HostChainRisksWarning({
+  hostChain,
+  riskCount,
+}: HostChainRisksWarningProps) {
+  const text = riskCount
+    ? 'There are ' + riskCount + ' risks coming from the hostchain - '
```
Comment (lucadonnoh) [inline packages/frontend/src/components/host-chain-risks-warning.tsx:15]:
```suggestion
    ? 'There are ' + riskCount + ' additional risks coming from the hostchain - '
```

**Review by lucadonnoh — DISMISSED** [review]

File: packages/frontend/src/components/host-chain-risks-warning.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+export function HostChainRisksWarning({
+  hostChain,
+  riskCount,
+}: HostChainRisksWarningProps) {
+  const text = riskCount
+    ? 'There are ' +
+      riskCount +
+      ' additional risks coming from the hostchain - '
```
Comment (torztomasz) [inline packages/frontend/src/components/host-chain-risks-warning.tsx:17]:
```suggestion
      ' additional risks coming from the hostchain '
```

File: packages/frontend/src/components/host-chain-risks-warning.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { type ScalingProjectDisplay } from '@l2beat/config'
+import Image from 'next/image'
+import Link from 'next/link'
+
+export type HostChainRisksWarningProps = {
+  hostChain: ScalingProjectDisplay
+  riskCount?: number
+}
+
+export function HostChainRisksWarning({
```
Comment (torztomasz) [inline packages/frontend/src/components/host-chain-risks-warning.tsx:10]: image
It's not centered

**Review by torztomasz — CHANGES_REQUESTED** [review]
Looks good, only the text needs to be centered in the warning

File: packages/frontend/src/components/host-chain-risks-warning.tsx (hunk 3 above)
Comment (sdlyy) [reply packages/frontend/src/components/host-chain-risks-warning.tsx:10]: https://github.com/l2beat/l2beat/pull/5318/commits/6932bfaa417b2a9306a5a3aa8f7749d1fb59eb22

File: packages/frontend/src/components/host-chain-risks-warning.tsx (hunk 2 above)
Comment (sdlyy) [reply packages/frontend/src/components/host-chain-risks-warning.tsx]: https://github.com/l2beat/l2beat/pull/5318/commits/6932bfaa417b2a9306a5a3aa8f7749d1fb59eb22

File: packages/frontend/src/components/host-chain-risks-warning.tsx (hunk 3 above)
Comment (sdlyy) [reply packages/frontend/src/components/host-chain-risks-warning.tsx:10]: https://github.com/l2beat/l2beat/pull/5318/commits/275a18de09e65285303be0988320b3a822a860b5

**Review by torztomasz — APPROVED** [review]

**Pull request merged**